roles/openshift_grafana/templates/setup-grafana.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
```

Comment:
I'm not sure I understand why this file was committed.

Reply:
it shouldn't I'll cleanup this one